common/sagas/contracts.js

```diff
@@ -0,0 +1,61 @@
+import { takeEvery, select, put } from 'redux-saga/effects';
+import type { Effect } from 'redux-saga/effects';
+import translate from 'translations';
+
+import {
+  setNodeContracts,
+  AccessContractAction,
+  setInteractiveContract
+} from 'actions/contracts';
+import { showNotification } from 'actions/notifications';
+import {
+  FETCH_NODE_CONTRACTS,
+  ACCESS_CONTRACT
+} from 'actions/contractsConstants';
+import { NODE_CHANGE } from 'actions/configConstants';
+import CONTRACTS from 'config/contracts';
+import { NODES } from 'config/data';
+import { getNode } from 'selectors/config';
+import { isValidETHAddress } from 'libs/validators';
+
+function* handleFetchNodeContracts() {
+  const nodeKey = yield select(getNode);
+  const node = NODES[nodeKey] || {};
+  const contracts = CONTRACTS[node.network] || [];
```

Comment:
we have `network`s, since we could have multiple nodes per network https://github.com/MyEtherWallet/MyEtherWallet/blob/develop/common/config/data.js#L136

you can get it via https://github.com/MyEtherWallet/MyEtherWallet/blob/develop/common/selectors/config.js#L11